Automattic/sensei
# Refactor accessibility on front-end module lessons expand button

Resolves #6649

## Proposed Changes
* Updates screen reader text to be less verbose.
* Adds `aria-expanded="true"` to capture default module state, expanded.

## Testing Instructions


1. Start a screen reader such as NVDA on Windows or Voiceover on Mac.
2. Create a course, modules, and lessons with example content.
3. View the course.
4. Notice how the button to expand/collapse a module now contains the module name instead of generic toggle module content text for screen readers.
5. Notice how `aria-expanded="true"` is now added to capture the initial state of the expanded module.

## New/Updated Hooks


*

## Deprecated Code


*

## Pre-Merge Checklist
